server/src/main/scala/drt/users/KeyCloakGroups.scala

```diff
@@ -0,0 +1,37 @@
+package drt.users
+
+import drt.shared.KeyCloakApi.{KeyCloakGroup, KeyCloakUser}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+
+case class KeyCloakGroups(groups: List[KeyCloakGroup]) {
+  def usersWithGroupsCsvContent(client: KeyCloakClient): Future[String] = {
+    val usersWithGroupsFuture = usersWithGroups(groups, client)
+    usersWithGroupsToCsv(usersWithGroupsFuture)
+  }
+
+  def usersWithGroupsToCsv(usersWithGroupsFuture: Future[List[(KeyCloakUser, String)]]): Future[String] = usersWithGroupsFuture
+    .map(_.groupBy { case (user, _) => user })
+    .map(usersToUsersWithGroups => {
+      val csvLines = usersToUsersWithGroups
+        .toSeq
+        .map {
+          case (user, usersWithGroups) =>
+            val groupsCsvValue = usersWithGroups.map(_._2).sorted.mkString(", ")
```

Comment:
clarify what _2 is by changing it to a case

Reply:
Done!